README.md

```diff
@@ -209,6 +209,24 @@ While it's an implementation detail, the new REST API uses many Spring REST (Jav
 
 Each of these libraries were chosen based on design principles above, and based on the fact that much of our underlying Java APIs use or align with Spring technologies.
 
+## Content negotiation
+The DSpace REST API only support the application/json response format at the current stage so there is no real content negotiation according to the broad meaning of the term.
+Nevertheless, some concepts of content negotiation still apply.
+For the POST requests the request content-type usually drive the creation of resource process according to different input format other than the preferred application/json such as text/uri-list and multipart/form-data. PUT requests on association endpoint accepts text/uri-list according to the Spring Data best practice.
+
+### Language supports
+While most i18n (internationalization) settings exist in the UI layer, there are some backend features which also require i18n, such as Submission forms, Controlled Vocabularies, Authorities. For such features the **Accept-Language** header can be used by the REST client to force the use of a specific locale, if supported by the DSpace instance and endpoint in the response. Please note that not all the dspace instances support multiple locales, this can be discovered querying the configuration endpoints for the _webui.supported.locales_ key.
+
+If the support for multiple locales is enabled in the DSpace instance, a request with an **Accept-Language** header is expected to be processed as follow
+* if none of the requested locales is supported the server will ignore the header to result more user friendly to whom  browses the REST endpoints via a browser (see https://tools.ietf.org/html/rfc7231#section-5.3.5)
+* if at least one acceptable locale is requested, the server will use the one with the higher priority in the Accept-Language header to process the request
+* if the requested endpoint is able to provide a different response for the requested locale compared to the other supported locales this will be noted in a Content-language Response Header otherwise such header will list all the supported locales. This also imply that the ROOT endpoint exposes, when such support is enabled, the supported locales via the **Content-Language** response header.
```

Comment:
The first sentence doesn't make sense to me here.  Are you saying that the `Content-Language` header is returned on *all requests*, even when `Accept-Language` is specified?  I'm not sure what this means:
> is able to provide a different response for the requested locale compared to the other supported locales

Reply:
The Content-Language is returned only when multiple locales are available.
If the current request has been served using a specific locale and there is a different response for other locales, the Content-Language will be the exact locale used to generate the response. If the response is the same for all the locales the Content-Language will be the list of supported locales. This allow the client to cache response in a more efficient way, for instance sa by now we don't expect to have different response for the `/api/core/items/:uuid` endpoint depending on the Accept-Language so the server will note that in the response say that the Content-Language is "en, it". If the user switch from the English UI to the Italian UI the rest request for the "item" is still valid.
On the other side, the submission form endpoint will respond with a specific Content-Language en or it so that the dc.title label can be Title or Titolo